src/bun.js/webcore/response.zig

```diff
@@ -1030,8 +1031,11 @@ pub const Fetch = struct {
                     }
 
                     if (options.get(ctx, "redirect")) |redirect_value| {
```

Comment:
```suggestion
                    if (options.getOptionalEnum(ctx, "redirect", FetchRedirect)) |redirect_value| {
```

Then, in `FetchRedirect`:

```zig
pub const Map = bun.enumMap(FetchRedirect, &.{ .{ "manual",  .manual}, .{"redirect", .redirect"}, .{"error", .error}});
```

then we can rename the enum values to their lower case version (except error would be `@"error"`)

This will handle printing a human-readable error message for us when the enum string value is incorrect (rather than silently ignoring it)

Reply:
enumMap might be wrong btw, can just use ComptimeStringMap

Reply:
fix here https://github.com/oven-sh/bun/commit/616bfbeef5a7877b327eacfe67ae5b65039d53c4